src/reducers/collection.ts

```diff
@@ -82,10 +82,10 @@ export function collection(
       return { ...state, recordsLoaded: false };
     }
     case COLLECTION_RECORDS_SUCCESS: {
-      const { records, hasNextRecords, listNextRecords } = action;
+      const { records, hasNextRecords, listNextRecords, isNextPage } = action;
       return {
         ...state,
-        records: [...state.records, ...records],
+        records: isNextPage ? [...state.records, ...records] : records,
```

Comment:
I'm not sure that I understand this. Maybe `isNextPage` does not represent what we are really doing here?
Could you please add a comment why we skip `state.records` when `isNextPage`?